sunpy/tests/setup_command.py

```diff
@@ -84,6 +84,12 @@ def _validate_required_deps(self):
                 "The 'test' command requires the sunpy package to be "
                 "installed and importable.")
 
+        doctestplus_specs = importlib.util.find_spec("pytest_doctestplus")
+        if doctestplus_specs is None or doctestplus_specs == 'namespace':
+            raise ImportError(
+                "The 'test' command requires the pytest-doctestplus to be "
+                "installed and importable.")
```

Comment:
Should change the sentence, to say that that `pytest-astropy` should be installed and that provides `pytest-doctestplus`. That is the module we have in the `dev.txt` file.

Reply:
Fixed it.